.env.docker

```diff
@@ -0,0 +1,10 @@
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+# Trello configuration
+TRELLO_BOARD_ID=64ae6f476f946f8c451a5cb2
+TRELLO_APIKEY=1539af5c1060361a7fe028400428d254
```

Comment:
Is there any reason you've chosen to use a different .env for docker rather than the one we've used locally? If it's intentional (e.g. using a separate board id to differentiate dev from prod) then we'll want to `.gitignore` this as well (and remove the existing committed file, with `git rm --cached .env.docker`)